src/test/regress/expected/window_functions_2.out

```diff
@@ -0,0 +1,1144 @@
+-- ===================================================================
```

Comment:
Not sure how we can deal with the differing order of sort keys? I'd suggested to @pykello that we could sort all the lines of a file when diffing

Reply:
I think the SortKey has changed because you've added an ORDER BY to the window function, right? So, shouldn't we simply update `window_functions.out` file instead of adding a new one?

Reply:
It changed because pg12 likes to do things in different orders than pg11 when the order doesn't matter. The query wasn't changed

Reply:
I think adding a new test output is an extreme action for a case where only the explain output has changed. The cost of maintaining extra files is really high.

I think we should find those queries and consider re-writing them in a way that they give consistent outputs. Because in most cases, we're really not interested in the explain output of the worker nodes/shards. If that's the case, lets just do this: https://github.com/citusdata/citus/pull/2793